src/components/ui/PopOver/index.tsx

```diff
@@ -44,6 +44,8 @@ export interface PopOverProps
   a11yLabel: string;
   /** The elements that populate the menu */
   children: any;
+  /** Unlike the default behavior, this prop allows the popover to stay open for multiple selections. */
+  multiSelect?: boolean;
```

Comment:
What are "selections" and should the `PopOver` component have direct knowledge of it?

I realize you mean clicks, but just pointing out that the prop name brings in assumptions about how this component is used. I would instead rename the prop to something that better describes what the prop is doing (which is preventing clicks from closing the popover)

Reply:
Ok, I thought about that. I just thought the prop name "dontImmediatelyClose" was bad but you are right.

Reply:
I think that `multiselect` is a bit of a loaded term. Usually that is related to `select` menus and the ability to choose more than one value. How about `closeOnClick` as a`boolean`?